hasura/migrations/default/1649169600501_alter_table_public_pending_token_gifts_add_unique_sender_id_recipient_id_epoch_id/down.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."pending_token_gifts" drop constraint "pending_token_gifts_sender_id_recipient_id_epoch_id_key";
```

Comment:
can we add the `IF EXISTS`?
```suggestion
alter table "public"."pending_token_gifts" drop constraint if exists "pending_token_gifts_sender_id_recipient_id_epoch_id_key";
```